recipes-ofdpa/ofdpa/ofdpa-platform/LICENSE.Broadcom-OpenBCM

```diff
@@ -0,0 +1,53 @@
+IMPORTANT-READ CAREFULLY: This license agreement ("Agreement") is a legal agreement between you (or if you are acting on behalf of your employer, then 'you' means your employer) ("Licensee") and Avago Technologies International Sales Pte. Limited, a Singapore corporation (with registration number 200512231E), with offices located at 1 Yishun Avenue 7, Singapore 768923 ("Broadcom"), a subsidiary of Broadcom Inc., for Licensee’s license to the Broadcom software that accompanies this Agreement which includes computer software (which may include source code, object code and/or software tools) and may include associated media, printed materials, datasheets, "online" or electronic documentation, and Internet-based services ("Software"). This Agreement is effective date as of the date in which you download, access, use, copy or install the Software ("Effective Date").
```

Comment:
In the commit message "ofdpa-platform: add a package for platform configs", there are two text pieces that need a bit of work:

- "as well a README.md describing the expected format"
- "as it the license of OpenBCM's custom_led.c"

Reply:
Fixed both of these and replaced the branch in the link with a commit hash to avoid the link getting broken in the future if Broadcom removes the SDK 6.5.24, as long as Broadcom does not rebase the repo.